src/test/regress/expected/multi_test_helpers_superuser.out

```diff
@@ -25,17 +25,8 @@ WITH dist_node_summary AS (
         master_run_on_worker(ARRAY[hostname, 'localhost'], ARRAY[port, master_port],
                             ARRAY[dist_node_summary.query, dist_node_summary.query],
                             false)
-), dist_placement_summary AS (
```

Comment:
hmmmm, is this change strictly necessary? I mean, not checking the placement metadata for a function like `verify_metadata` sounds not expected. 

I wonder after all the improvements on your PR, would this work again? In the end, we clear all the metadata during re-activate?